nsxt/resource_nsxt_policy_l2_vpn_session_test.go

```diff
@@ -142,7 +142,7 @@ func TestAccResourceNsxtPolicyL2VpnSession_tier1(t *testing.T) {
 		},
```

Comment:
any reason this test is only running for versions below 3.2.0?

Reply:
It should be "version higher than 3.2.0", now fixed